get-started/index.md

```diff
@@ -177,22 +177,54 @@ Once you have authenticated through the cluster's SSO login you will be
 redirected back to the Cryostat web application. The redirect URL contains
 an access token for Cryostat's service account with the permissions you have
 granted to it. This access token will eventually expire and you will be
```

Comment:
> This access token will eventually expire and you will be required to log back in on the cluster SSO login page. 

Can this line comes last in the paragraph so that it sorta establish a better flow from creation -> usage -> renew?